app/Models/Category.php

```diff
@@ -84,7 +84,8 @@ public function _id($id) {
 		}
 	}
 	public function _name($value) {
-		$this->name = trim($value);
+		// regression of b323ed084?
+		$this->name = mb_strcut(trim($value), 0, 255, 'UTF-8');
```

Comment:
@Alkarex I need a check here because of https://github.com/FreshRSS/FreshRSS/commit/b323ed084#diff-4618a3f4bd008df547c9bbc41e2bd145L71-R71 I don't see why it was changed and it doesn't seem to be related to the commit

Reply:
Indeed, that looks like a mistake